test/controllers/checklist_template_items_controller_test.rb

```diff
@@ -6,13 +6,13 @@ class ChecklistTemplateItemsControllerTest < ActionDispatch::IntegrationTest
   end
 
   test "should get index" do
-    get checklist_template_items_url
-    assert_response :success
+    # get checklist_template_items_url
+    # assert_response :success
```

Comment:
Could you use use the `skip` method here so we can see which ones we skipped in the minitest output? Otherwise I think it can get buried :P